src/irc.nim

```diff
@@ -33,11 +33,13 @@ import net, strutils, strtabs, parseutils, times, asyncdispatch, asyncnet
 import os, tables, deques
 from nativesockets import Port
 export `[]`
-
+export Port
 type
   IrcBaseObj*[SockType] = object
     address: string
     port: Port
+    proxyAddr: string
+    proxyPort: Port
```

Comment:
So I assume that this is a socks proxy, change all `proxy` to `socksProxy`. Not just here, but below as well.
```suggestion
    socksProxyAddr: string
    socksProxyPort: Port
```